proposals/dao/tip_121c.ts

```diff
@@ -63,6 +93,22 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
   console.log(' Equity diff                            [M]e18 ', Number(eqDiff) / 1e24);
   console.log('----------------------------------------------------');
 
+  // Calculate protocol controlled Fei and user circulating - subtracting off the 10M FEI
+  // our test scripts mint. The use to determin if over collaterised
+  const protocolControlledFei = (await contracts.fei.balanceOf(addresses.simpleFeiDaiPSM))
+    .add(await contracts.fei.balanceOf(addresses.rariInfraFeiTimelock))
+    .add(await contracts.rariPool79FeiPCVDepositWrapper.balance());
+
+  const userCirculatingFeiSupply = (await contracts.fei.totalSupply())
```

Comment:
Please verify this logic which checks that we are overcollaterised. I'm subtracting off the 10M FEI our e2e setup script mints - the on-chain CR here will have it factored into user circ Fei and so report as undercollaterised

Reply:
Looks right to me, should verify against a recent fork block